windows_create_package.bat

```diff
@@ -41,13 +43,15 @@ if !errorlevel! neq 0 exit /b !errorlevel!
 
 :: Prepare self-contained python
 set "dest_dir=C:\opt"
-set "python_version=3.9.13"
+set "python_version=3.11.9"
+
 call %cd%\windows_prepare_python.bat %dest_dir% %python_version%
 :: Copy whole catalog to dist folder and install dependencies required by LLM pipelines
 xcopy %dest_dir%\python-%python_version%-embed-amd64 dist\windows\ovms\python /E /I /H
 .\dist\windows\ovms\python\python.exe -m pip install "Jinja2==3.1.4" "MarkupSafe==3.0.2"
```

Comment:
Don't we need separate `if !errorlevel! neq 0 ...` parts for the `call` and `xcopy` here?